doc-site/components/nve-alert.md

```diff
@@ -1,127 +1,117 @@
 ---
 layout: component
+outline: [2, 3]
 ---
 
 <CodeExamplePreview>
 
 ```html
-<nve-alert open text="text"></nve-alert>
+<nve-alert open label="Info" text="Dette er alert komponent. Den har ventet hele livet på å varsle deg!"></nve-alert>
 ```
 
 </CodeExamplePreview>
 
-<nve-message-card label="Tips">
-Bruk `open` for å vise en alert. Hvis ikke `open` er satt, vises den ikke.
+<nve-message-card label="Tips" size="compact">
+<p>Bruk <b>open</b> for å vise en alert. Hvis ikke <b>open</b> er satt, vises den ikke.</p>
 </nve-message-card>
 
 ## Eksempler
 
-### Variant
+### Styr innhold
 
-Bruk `variant` for å styre farger. `primary` er standard og trenger ikke spesifiseres.
+Du kan bruke spor for å vise innhold i alerten. Standard-spor er hovedteksten. Du kan også bruke egne spor for ikon (icon) og overskrift (label).
 
-<CodeExamplePreview arrangeComponentsVertically>
+<CodeExamplePreview>
 
 ```html
 <nve-alert open>
   <nve-icon slot="icon" name="info"></nve-icon>
-  primary
+  <nve-label slot="label">Label</nve-label>
+  Bruk av slots i alerten.
 </nve-alert>
+```
 
-<nve-alert variant="success" open>
-  <nve-icon slot="icon" name="check_circle"></nve-icon>
-  success
-</nve-alert>
+</CodeExamplePreview>
 
-<nve-alert variant="warning" open>
-  <nve-icon slot="icon" name="warning"></nve-icon>
-  warning
-</nve-alert>
+Som alternativ kan du bruke attributer iconName, label og text.
 
-<nve-alert variant="neutral" open>
-  <nve-icon slot="icon" name="help"></nve-icon>
-  neutral
-</nve-alert>
+<CodeExamplePreview>
 
-<nve-alert variant="danger" open>
-  <nve-icon slot="icon" name="error"></nve-icon>
-  danger
-</nve-alert>
+```html
+<nve-alert open label="Label" text="Bruk av attributer"></nve-alert>
 ```
 
 </CodeExamplePreview>
 
-### Left stroke
+### Variant
 
-Bruk `leftStroke` for å vise en fet strek på venstre side.
+Bruk `variant` for å styre farger. `primary` er standard og trenger ikke spesifiseres.
 
 <CodeExamplePreview arrangeComponentsVertically>
 
 ```html
-<nve-alert variant="danger" text="Left stroke" leftStroke open> </nve-alert>
+<nve-alert open text="primary"></nve-alert>
+
+<nve-alert variant="success" open text="success"></nve-alert>
+
+<nve-alert variant="warning" open text="warning"></nve-alert>
+
+<nve-alert variant="neutral" open text="neutral"></nve-alert>
+
+<nve-alert variant="danger" open text="danger"></nve-alert>
 ```
 
 </CodeExamplePreview>
 
-### Høyere metning
+### Ikon
 
-Bruk `saturation="emphasized"` for å få litt mørkere bakgrunnsfarge.
+Velger du variant kommer det en bestemt ikone som standard. Du kan overstyre ikone med `iconName` property.
```

Comment:
Fint med et eget avsnitt for dette!
Ikon staves uten e til slutt. ikon er intetkjønn, altså "et ikon", ikke "en ikon". Sjekk staving og grammatikk i hele dette Ikon-avsnittet